src/test/java/com/corpo/coliseum/domain/nrt/CorporationIntegrationTest.java

```diff
@@ -0,0 +1,4 @@
+package com.corpo.coliseum.domain.nrt;
+
+public class CorporationIntegrationTest {
```

Comment:
why initiate a class that does nothing?
if it is preparation for integration tests, you might as well do it at the time of implementation